### ISSUE TITLE
Automatically accept first share

### ISSUE THREAD

**Issue by PVince81** [owncloud/guests#323]
### Steps

1. As admin unset "Automatically accept local shares"
1. Invite a guest
1. Login as guest

### Expected result
Share still appears directly, especially if it's the first one.

### Actual result
"Where is the share ??"
Need to click on the notification icon to accept it first. This feels a bit redundant.

### Version
Guests 0.8.1 RC1

@pmaier1

**Comment by pmaier1** [comment]
Yes, I noticed this as well. Users that are new to oC will be confused by this and there's no sense in not auto-accepting the first share in this case. Should be changed as suggested.
